frontend/src/apis/sales/SalesConversions.tsx

```diff
@@ -40,5 +44,25 @@ export function APIToInternalSRConversion(sr: SalesAPI.APISR): SalesRecord {
     totalRevenue: sr.total_revenue,
     sales: sales,
     isDeletable: sr.is_deletable,
+    isSalesReconciliation: sr.is_reconciliation,
+    creatorName: sr.username,
   };
 }
+
+export function APIToInternalSalesCSVConversion(
+  sales: APISaleCSVImportRow[]
+): LineItem[] {
+  return sales.map((sale) => {
+    return {
+      isNewRow: true,
+      id: uuid(),
+      subtotal: 0,
+      bookId: sale.book,
+      bookTitle: formatBookForDropdown(sale.book_title, sale.isbn_13),
+      bookISBN: sale.isbn_13,
+      quantity: sale.quantity,
+      price: sale.unit_retail_price,
+      errors: sale.errors,
```

Comment:
I think this is actually csvErrors (check the line item data type to confirm, I'm not 100% sure), there was a bug involving this before